code/lib/test/src/expect.ts

```diff
@@ -16,9 +16,10 @@ import {
 } from '@vitest/expect';
 import * as matchers from '@testing-library/jest-dom/matchers';
 import type { PromisifyObject } from './utils';
+import type { TestingLibraryMatchers } from '@testing-library/jest-dom/types/matchers';
```

Comment:
This line seems to be translated as 
`import { TestingLibraryMatchers } from '@testing-library/jest-dom/types/matchers';` 
in `dist/index.d.ts`, which typescript can't parse right and causes it to be typed as `any`. 

Can you replace this with 
`import type { TestingLibraryMatchers } from '@testing-library/jest-dom/matchers'` 
by any chance?